docs/runtime-contract.md

```diff
@@ -482,28 +448,7 @@ From the OCI spec:
 > article in the kernel documentation has more information about mount
 > propagation.
 
-This option should only be set by the operator or platform provider, and MUST
-NOT be configurable by the developer. As mount propagation may be part of the
+This option MAY be set by the operator or platform provider, and MUST
+NOT be configurable by the developer. As mount propagation MAY be part of the
 platform security hardening, operators MAY tune this over time as the threat
 environment changes.
-
-### Masked Paths
-
-This option MAY only be set by the operator or platform provider, and MUST NOT
-be configurable by the developer. As masked paths may be part of the platform
-security hardening, operators may tune this from time to time as the threat
-environment changes.
```

Comment:
Again, wondering if it's worth documenting where the responsibility for these setting lies, given that they are in the OCI spec.

Reply:
This section seemed overly limiting and does not seem to prevent the operation of Knative. Why must this not be configurable by the developer? It seemed to me like you could let the developer have control over some of these things. Given that the OCI interface should not be available within the container the only way to configure this would be through the Knative API. An extension of the Knative API to allow certain paths to be masked or read-only didn't seem crazy to me.

Reply:
I'm fine with dropping this.